zips/zip-0315.rst

```diff
@@ -170,6 +170,30 @@ in any case due to padding of the number of inputs from a given
 shielded pool.
 
 
+Address management
+------------------
+
+Zcash wallets SHOULD only use addresses generated deterministically from a
+seed as specified in ZIP 32 [#zip-0032]_. The discussion of address management
+below assumes this is the case.
```

Comment:
This section will need to be updated for FROST. For now it is correct as a SHOULD.